lib/actions/ui.js

```diff
@@ -32,6 +33,9 @@ import {
 import {setActiveGroup} from './term-groups';
 
 const {stat} = window.require('fs');
+const isWin = process.platform === 'win32';
+const plugins = remote.require('./plugins');
+const {shell} = plugins.getDecoratedConfig();
```

Comment:
you need to move this inside `getEditCommand` or `showPreferences`
Otherwise it will just be set at import time, and when you use this after having edited the settings, this variable won't update 😄